sdk/identity/identity/review/identity.api.md

```diff
@@ -5,10 +5,13 @@
 ```ts
 
 import { AccessToken } from '@azure/core-http';
+import { AccessToken as AccessToken_2 } from '@azure/core-auth';
```

Comment:
this is supposed to be the same type, just `core-http` re-exported it, right? Can we import consistently from one of the packages instead of both?

Reply:
I fixed both of these by replacing all of them with the core-auth variant. Hopefully that'll make it easier to move to core-rest-pipeline in the future.